src/libraries/System.Net.Sockets/src/System/Net/Sockets/Socket.cs

```diff
@@ -3816,7 +3816,7 @@ public bool AcceptAsync(SocketAsyncEventArgs e)
             }
             if (e.HasMultipleBuffers)
             {
-                throw new ArgumentException(SR.net_multibuffernotsupported, "BufferList");
+                throw new ArgumentException(SR.net_multibuffernotsupported, nameof(e));
```

Comment:
This is another example of a case where perfect is the enemy of the good.  How does someone know in this case that they're getting an ArgumentException because e.BufferList was invalid?  They used to get an exception that called it out "Param name: BufferList"... now they'll just get "Param name: e".  Such changes are making the developer experience worse, IMHO.

Reply:
Ah, I see... the exception message itself calls out multiple buffers.  Ok, that's reasonable.